lib/routes/validators.js

```diff
@@ -153,3 +153,7 @@ module.exports.isValidUrl = function (redirect, regex) {
   var parsed = url.parse(redirect)
   return regex.test(parsed.hostname) && /^https?:$/.test(parsed.protocol)
 }
+
+module.exports.verificationMethod = function() {
+  return  isA.string().valid(['email', 'email-2fa', 'email-captcha'])
+}
```

Comment:
Given that this function doesn't take any args, it could also be exported as a static property. And if we did that, I *think* the doc-generator script would then automatically expand `verificationMethod` in the docs. (and if it doesn't, it will be straightforward to fix because we have that working for metrics context stuff)